docs/markdown/Continuous-Integration.md

```diff
@@ -106,45 +189,28 @@ AppVeyor also has
 [MacOS](https://www.appveyor.com/docs/macos-images-software/) and
 [Linux](https://www.appveyor.com/docs/linux-images-software/) CI
 images. This is a sample `appveyor.yml` file for Windows with Visual
-Studio 2015 and 2017.
+Studio 2017, 2019, and 2022.
 
 ```yaml
-image: Visual Studio 2017
-
+version: 1.0.{build}
+image:
+- Visual Studio 2022
+- Visual Studio 2019
+- Visual Studio 2017
 environment:
-  matrix:
-    - arch: x86
-      compiler: msvc2015
-    - arch: x64
-      compiler: msvc2015
-    - arch: x86
-      compiler: msvc2017
-    - arch: x64
-      compiler: msvc2017
-
-platform:
-  - x64
+  _CRT_SECURE_NO_WARNINGS: 1
+  _SCL_SECURE_NO_WARNINGS: 1
```

Comment:
These should not be on by default as they might hide safety issues. They should be commented out with a comment saying that if you are willing to accept the risk, you can uncomment the following lines.